mne/channels/channels.py

```diff
@@ -1335,3 +1335,63 @@ def _get_ch_info(info):
     return (has_vv_mag, has_vv_grad, is_old_vv, has_4D_mag, ctf_other_types,
             has_CTF_grad, n_kit_grads, has_any_meg, has_eeg_coils,
             has_eeg_coils_and_meg, has_eeg_coils_only)
+
+
+def create_1020_montage_mapping(ch_names, midline="z12"):
+    r"""Return dict mapping from ROI names to lists of picks for 10/20 setups.
```

Comment:
Why is a raw string necessary?

Reply:
I thought we were defaulting to that ...

Reply:
Yikes! Why?

Reply:
Idk, maybe I misread something. @larsoner ?

Reply:
I don't think we need `r''` as a default. Usually it's useful when you will have a lot of `\` in the string like when using math mode or regular expressions.